src/args.c

```diff
@@ -55,6 +55,9 @@ void args_parse(int argc, char **argv)
 			config.position_mode = POSITION_MODE_CENTER;
 		} else if (!strncmp(argv[i], "--persistent", 12)) {
 			config.persistent = 1;
+		} else if (override && !strncmp(argv[i], "--override=", 11)) {
```

Comment:
Suggest we create a copy to avoid an complication associated with `config_process_line()` modifying the data.

```
diff --git a/src/args.c b/src/args.c
index b185083..9304296 100644
--- a/src/args.c
+++ b/src/args.c
@@ -56,8 +56,13 @@ void args_parse(int argc, char **argv, int override)
 		} else if (!strncmp(argv[i], "--persistent", 12)) {
 			config.persistent = 1;
 		} else if (override && !strncmp(argv[i], "--override=", 11)) {
-			// modifies argv, should only be called once
-			config_process_line(argv[i] + 11);
+			/*
+			 * Create a copy as config_process_line modifies the
+			 * C string passed to it; and we call this more than once.
+			 */
+			char *line = xstrdup(argv[i] + 11);
+			config_process_line(line);
+			free(line);
 		}
 	}
 }
```